assembl/static2/scripts/translations2messages.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const translations = require('../js/app/utils/translations');
+
+const englishTranslations = translations.en;
+
+function flatten(arr) {
```

Comment:
we have `lodash` in the dependencies, maybe we could use https://lodash.com/docs/4.17.4#flatten ?